news/2018-04-23-project-loved-week-of-april-22nd.md

```diff
@@ -0,0 +1,287 @@
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/XY-hIKbuoF8?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+---
+
+[![](/wiki/shared/news/2018-04-23-project-loved-week-of-april-22nd/mania/scrap-syndrome.jpg)](https://osu.ppy.sh/community/forums/topics/736018)
+
+[Yuuyu - Scrap Syndrome](https://osu.ppy.sh/beatmapsets/663256) by [Halogen-](https://osu.ppy.sh/users/169992)  
+*written by [Shoegazer](https://osu.ppy.sh/users/2520707)*
+
+Low-key one of the best charters in osu!mania and resident sadist [Halogen-](https://osu.ppy.sh/users/169992) gets another merciless chart onto the Loved nomination table with [Scrap Syndrome](https://osu.ppy.sh/beatmapsets/663256). While this chart was made specifically as an mixed pick for the finals stage of MWC 4K 2017, the chart itself stands out as one of the best, if not THE best hyper-technical and all-rounded charts in osu!mania. 
+
+Fantastic use of repetition, (primarily) SVs that accentuate the track than overtake it, meticulous layering and patterning structure, sparing but calculating use of LNs, they are all there. The primary complaint you can make about the chart is that it can be a bit too sadistic in structure, and I would agree with it to some degree, but the cruelty is by design -- and it works **brilliantly**. 
+
+While osu!mania has had this shift in charting meta to something more cutting-edge and flashy like SV-heavy charts, LN charts, and super-technical charts, I feel that they tend to march to the beat of their own drums; they end up overshadowing the track itself for charts test specific skills. Scrap Syndrome stands out for not only being the apex of that charting meta, but also bringing a finesse of musical relevance and cutting-edge approach that is almost never seen in charting today. 
+
+In my eyes, and in the eyes of many, this chart is more than qualified to be nominated into the Loved section. There may be more flashy and impressively built charts than Scrap Syndrome, but very few charts do a job of capturing the track while engaging players better than Scrap Syndrome.
+
+[![](/wiki/shared/news/2018-04-23-project-loved-week-of-april-22nd/mania/takecore.jpg)](https://osu.ppy.sh/community/forums/topics/736019)
+
+[DJ Sharpnel - TAKECORE OF YOURSELF](https://osu.ppy.sh/beatmapsets/480479) by [Wh1teh](https://osu.ppy.sh/users/2640467)  
+*written by [Shoegazer](https://osu.ppy.sh/users/2520707)*
+
+A favourite among competitive circles, [TAKECORE OF YOURSELF](https://osu.ppy.sh/beatmapsets/480479) is among one of [Wh1teh](https://osu.ppy.sh/users/2640467)'s most played charts in osu!mania. Given the type of chart that TAKECORE OF YOURSELF is, it is no surprise that it is so popular. With clear inspirations from IcyWorld's DJ SHARPNEL stamina-intensive charts, Wh1teh takes it a step further and adds patterning nuance with this chart to capture the ethos of the accompanying track further. 
```

Comment:
[TAKECORE OF YOURSELF](https://osu.ppy.sh/beatmapsets/480479) missing
https://osu.ppy.sh/users/1383951 IcyWorld Link
[Wh1teh](https://osu.ppy.sh/users/2640467) wh1teh link